projects/admin/src/app/circulation/patron/history/history-item/history-item.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { extractIdOnRef, RecordService } from '@rero/ng-core';
+import { forkJoin } from 'rxjs';
+import { ProvisionActivityType } from '../../../../pipe/provision-activity.pipe';
+import { MainTitleService } from '../../../../service/main-title.service';
+import { PatronService } from '../../../../service/patron.service';
+
+@Component({
+  selector: 'admin-history-item',
+  templateUrl: './history-item.component.html'
+})
+export class HistoryItemComponent implements OnInit {
+
+  /** loan to display */
+  @Input() loan: any;
+
+  /** Item linked to the loan */
+  item: any;
+
+  /** Document linked to the loan */
+  document: any;
+
+  /** is loading */
+  isLoading = false;
+
+  /** Is collapsed */
+  isCollapsed = true;
+
+  /** ProvisionActivityType reference */
+  provisionActivityType = ProvisionActivityType;
+
+  /**
+   * Constructor
+   * @param _recordService - RecordService
+   * @param _patronService - PatronService
+   */
+  constructor(
+    private _recordService: RecordService,
+    private _patronService: PatronService,
+    private _mainTitleService: MainTitleService
+  ) { }
+
+  /**
+   * Component initialization.
+   * Load additional informations related to the loan : item and document
+   */
+  ngOnInit() {
+    if (this.loan) {
+      this._recordService.getRecord('items', this.loan.metadata.item_pid).subscribe(
+        (result) => {
+          const documentId = extractIdOnRef(result.metadata.document.$ref);
+          const itemObservable = this._patronService.getItem(result.metadata.barcode);
+          const documentObservable = this._recordService.getRecord(
+            'documents', documentId, 1,
+            { Accept: 'application/rero+json, application/json'}
+          );
+          forkJoin([itemObservable, documentObservable]).subscribe(
+            (results) => {
+              this.item = results[0];
+              this.document = results[1];
+            }
+          );
+        });
+    }
+  }
+
+  /**
+   * Get main title (corresponding to 'bf_Title' type, present only once in metadata)
```

Comment:
Please add **@return** for each method.
Doc blocks must be written as this (empty line before and after method description)

```
/**
 * Description of the method
 *  @param describe parameters
 * @return describe what is returned
 */
```

Reply:
Done.
Juste using the same method description used into other `getMainTitle` methods from other components